README.md

```diff
@@ -64,7 +64,7 @@ _Many of the links above are to Model View Culture's issue on events. I have lin
 * [TXJS](http://texasjavascript.com) // [@txjs](https://twitter.com/txjs)
 * [XOXO](http://2014.xoxofest.com/) // [@xoxox](https://twitter.com/xoxo)
 
-####New York
+###New York
```

Comment:
This should be four - it's part of North America. I only really have it because I was living there. Can you change it back to 4 hashtags?